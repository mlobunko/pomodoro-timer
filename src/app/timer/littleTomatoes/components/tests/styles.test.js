import React from "react";
import { shallow } from "enzyme";
import "enzyme-to-json";

import {
  StyledAnimatedLittleTomato,
  StyledLittleTomato,
  StyledLittleTomatoesComponent
} from "../styles";

describe("StyledAnimatedLittleTomato", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledAnimatedLittleTomato />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledLittleTomato", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledLittleTomato />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledLittleTomatoesComponent", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledLittleTomatoesComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
