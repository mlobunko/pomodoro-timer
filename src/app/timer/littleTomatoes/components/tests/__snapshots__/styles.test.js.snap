// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledAnimatedLittleTomato renders correctly 1`] = `
<div
  className="sc-bdVaJa hsdkLN"
/>
`;

exports[`StyledLittleTomato renders correctly 1`] = `
<div
  className="sc-bwzfXH ksyBEl"
/>
`;

exports[`StyledLittleTomatoesComponent renders correctly 1`] = `
<div
  className="sc-htpNat dWHtTZ"
/>
`;
